drivers/clock_control/clock_control_mcux_syscon.c

```diff
@@ -27,7 +27,7 @@ static int mcux_lpc_syscon_clock_control_on(const struct device *dev,
 	if ((uint32_t)sub_system == MCUX_MRT_CLK) {
 #if defined(CONFIG_SOC_FAMILY_LPC)
 		CLOCK_EnableClock(kCLOCK_Mrt);
-#elif defined(CONFIG_SOC_FAMILY_IMX)
+#elif defined(SOC_FAMILY_NXP_IMXRT)
```

Comment:
I'm not sure I follow here- I'm using this Kconfig because (to my knowledge) the SYSCON block is only present on LPC and NXP iMX RT parts, so this config should be enough to determine when to use the given `CLOCK` API